utils/vscode/package.json

```diff
@@ -3,21 +3,27 @@
   "displayName": "carbon-lang",
   "description": "Carbon Language support",
   "version": "0.0.2",
+  "publisher": "carbon-language",
```

Comment:
If I understand correctly, this field is supposed to be a unique identifier that's registered with Visual Studio Code Marketplace, and belongs to the Azure DevOps organization publishing the extension. I don't think we're ready to set up that kind of account-management infrastructure, so for now I think we should continue to leave this field out, and not publish the extension in the marketplace.

Reply:
Without publisher we can't actually create vsix package, I will change it if you wish